tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -3000,9 +3017,15 @@ public function userHasStoredEtagOfElementFromSpace(string $user, string $path,
 	/**
 	 * @Given /^user "([^"]*)" has stored etag of element "([^"]*)" on path "([^"]*)" inside space "([^"]*)"$/
 	 *
+	 * @param $user
+	 * @param $path
+	 * @param $storePath
+	 * @param $space
+	 *
+	 * @return void
 	 * @throws Exception | GuzzleException
 	 */
-	public function userHasStoredEtagOfElementOnPathFromSpace($user, $path, $storePath, $space) {
+	public function userHasStoredEtagOfElementOnPathFromSpace($user, $path, $storePath, $space):void {
```

Comment:
Note: we can also declare parameter types. That is done in most of the functions here, but is missing from this function.

@SagarGi please get someone to look through the PHP test code and add type declarations wherever we can.